src/main/java/seedu/address/ui/MainWindow.java

```diff
@@ -199,8 +199,6 @@ private CommandResult executeCommand(String commandText) throws CommandException
             if (commandText.startsWith(ListWeddingCommand.COMMAND_WORD)) {
                 weddingListPanel = new WeddingListPanel(logic.getFilteredWeddingList());
                 listPanelPlaceholder.getChildren().add(weddingListPanel.getRoot());
-            } else if (commandText.startsWith("view-wed")) {
-                // to fill
             } else {
```

Comment:
Good job differentiating the personListPanel & weddingListPanel and putting it in the appropriate place!